### ISSUE TITLE
`vendor\bin\deptrac init` doesn't work

### ISSUE THREAD

**Issue by roxblnfk** [qossmic/deptrac-shim#8]
```
Failed to copy "phar://D:/git/spiral/app/vendor/qossmic/deptrac-shim/deptrac/config/deptrac_template.yaml" because file does not exist.
```

OS: Windows 10

**Comment by dbrumann** [comment]
Hey, thanks for the report. Sorry for taking so long to reply, but I rarely check the issues here. We use qossmic/deptrac instead. I will take a look at this issue soon

**Comment by dbrumann** [comment]
The problem should be fixed now with the new 1.0.1 release
